gradle/scala.gradle

```diff
@@ -49,12 +49,18 @@ test {
 sourceSets {
     main {
         scala {
-            srcDir "src/main/scala_$scalaBaseVersion"
+            srcDirs += ["src/main/scala_$scalaBaseVersion", "src/main/java"]
+        }
+        java {
+            srcDirs = []
         }
     }
```

Comment:
ahh, i see
so maybe at the end is better to keep java files in scala directory and do not move them. 
Nasty, but easy solution